c10/core/DispatchKey.cpp

```diff
@@ -137,6 +137,9 @@ const char* toString(DispatchKey t) {
     case DispatchKey::CompositeExplicitAutograd:
       return "CompositeExplicitAutograd";
 
+    case DispatchKey::MetaInit:
+      return "MetaInit";
+
```

Comment:
@bdhirsh is working on extending the number of dispatch keys PyTorch can have.

If you need something sooner, maybe you can delete one of the PrivateUse keys, I don't know if anyone else uses them

Reply:
Obviously deleting one of the PrivateUse keys would be the fastest and easiest approach. However I assume that would mean a BC-breaking change. If @bdhirsh's work is targeted for v1.11, I can temporarily delete PrivateUse3 and later reintroduce it before the v1.11 release. @bdhirsh, @albanD  I would also appreciate your feedback on this.

Reply:
These keys are used by third party repos outside of core. It is hard to know the impact of deleting one so I don't think we should.

Reply:
@bdhirsh any estimate on when your work will be ready to land?

Reply:
Likely some time in the beginning of next year (January?) If you're interested in tracking progress, the prototype I'm working on is here: https://github.com/pytorch/pytorch/pull/68744